cni/plugin.go

```diff
@@ -177,3 +151,39 @@ func (plugin *Plugin) Error(err error) *cniTypes.Error {
 func (plugin *Plugin) Errorf(format string, args ...interface{}) *cniTypes.Error {
 	return plugin.Error(fmt.Errorf(format, args...))
 }
+
+// Initialize key-value store
+func (plugin *Plugin) InitializeKeyValueStore(config *common.PluginConfig) error {
```

Comment:
Why do we need this? We are already saving the state of networkManager. What happens if we get a GET call before any ADD happens? Even in this case I don't think we need to initialize json store here, as we can print an empty result.

Reply:
This initialize is called from the main. We can either check for the operation type in main and then avoid initialization, or we can always initialize and do the operation to keep logic simple. i'm fine with either.